spec/data/spec.yamlld

```diff
@@ -14,6 +14,9 @@ license:
   $id: https://purl.org/NET/rdflicense/W3C1.0
   dcterms:isReferencedBy: https://purl.org/NET/rdflicense/W3C1.0.ttl
 
+dcterms:isReferencedBy:
```

Comment:
is this the appropriate predicate?
I don't see any reference to the YAML-LD spec in namespace-prefixes.yamld...

Reply:
Makes sense. What about `dcterms:requires`?

Reply:
I don't think that the spec *requires* the prefix definitions... They are used in examples only.

If anything, dcterms:references would work, IMO.

Reply:
Done :white_check_mark: